.github/workflows/ci.yml

```diff
@@ -28,11 +28,14 @@ jobs:
         key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
         restore-keys: ${{ runner.os }}-composer-
 
+    - name: Check PHP syntax
+      run: composer syntax
+
     - name: Check Composer lock file is up to date
       run: composer validate --no-check-all
 
     - name: Install Composer dependencies
       run: composer install --no-progress --prefer-dist --optimize-autoloader
 
     - name: Run tests
-      run: composer test
```

Comment:
Could you please update https://github.com/MITLibraries/mitlib-wp-network/blob/master/.github/README.md?plain=1#L140-L144 with this new setup?